.github/workflows/test-package.yml

```diff
@@ -5,30 +5,25 @@ on: [ push, pull_request ]
 jobs:
   test:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}-latest
     defaults:
       run:
         working-directory: 'go/src/github.com/bugsnag/panicwrap' # relative to $GITHUB_WORKSPACE
     strategy:
       fail-fast: false
       matrix:
-        go-version: ['1.7', '1.8', '1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15']
```

Comment:
Changed this line since the support window is currently 1.9+ anyway, and to be able to use the `setup-go` action which can be used to install go on every platform.